internal/profile/profile_json.go

```diff
@@ -0,0 +1,55 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package profile
+
+import (
+	"encoding/json"
+	"os/user"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/version"
+)
+
+// Metadata stores the data associated with a given profile
+type Metadata struct {
```

Comment:
I looked in the `profile.json` file:

```
{
  "Name": "default",
  "date_created": "2021-05-14T11:03:48.44578+02:00",
  "User": "marcin.tojek",
  "elastic_package_version": "8050175",
  "From": "/Users/marcin.tojek/.elastic-package/stack/default"
```

Would you mind adjusting all fields to be lower cased (consistency)?

Reply:
good catch!